src/modules/LanItems.rb

```diff
@@ -2018,21 +2029,24 @@ def Commit
         newdev["INTERFACETYPE"] = @type
       end
 
-      NetworkInterfaces.Name = Ops.get_string(@Items, [@current, "ifcfg"], "")
-      NetworkInterfaces.Current = deep_copy(newdev)
+      if GetCurrentMap() != newdev
```

Comment:
It can happen that ```GetCurrentMap``` is undocumented. It is old method from YCP times.

Relationship is almost none. This part of ```Commit``` stores new devmap which is built from various "global" variables spread over the ```LanItems```. ```Items``` just provides proper ifcfg name here.